app/ts-cli/geminicli/export_test.go

```diff
@@ -0,0 +1,862 @@
+/*
+Copyright 2022 Huawei Cloud Computing Technologies Co., Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package geminicli
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"github.com/openGemini/openGemini/engine"
+	"github.com/openGemini/openGemini/engine/index/tsi"
+	"github.com/openGemini/openGemini/lib/netstorage"
+	"github.com/openGemini/openGemini/lib/util"
+	"github.com/openGemini/openGemini/open_src/influx/meta"
+	"github.com/openGemini/openGemini/open_src/vm/protoparser/influx"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"log"
+	"math"
+	"math/rand"
+	"os"
+	"path"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+var mockDbDiskInfo DatabaseDiskInfo = DatabaseDiskInfo{
+	dbName: "test_db2",
+	rps: map[string]struct{}{"test_db2_rp1": struct {
+	}{}, "test_db2_rp2": struct {
+	}{}},
+	dataDir: "/tmp/og_test/data/data/test_db2",
+	walDir:  "/tmp/og_test/wal/wal/test_db2",
+	rpToTsspDirMap: map[string]string{
+		"test_db2_rp1": "/tmp/og_test/data/data/test_db2/0/test_db2_rp1",
+		"test_db2_rp2": "/tmp/og_test/data/data/test_db2/0/test_db2_rp2",
+	},
+	rpToWalDirMap: map[string]string{
+		"test_db2_rp1": "/tmp/og_test/wal/wal/test_db2/0/test_db2_rp1",
+		"test_db2_rp2": "/tmp/og_test/wal/wal/test_db2/0/test_db2_rp2",
+	},
+	rpToIndexDirMap: map[string]string{
+		"test_db2_rp1": "/tmp/og_test/data/data/test_db2/0/test_db2_rp1/index",
+		"test_db2_rp2": "/tmp/og_test/data/data/test_db2/0/test_db2_rp2/index",
+	},
+}
+
+func createTestDirs() {
+	err := os.MkdirAll("/tmp/og_test/data/data/test_db1/0/test_db1_rp1/1_1568592000000000000_1569196800000000000_1", os.ModePerm)
```

Comment:
use `tmpDir := t.TempDir()` to generate temp directory.

Reply:
refactored.